src/aurora/pefile.cpp

```diff
@@ -143,7 +143,7 @@ void PEFile::load(const std::vector<Common::UString> &remap) {
 		if (id >= remap.size())
 			res.name = Common::UString::format("cursor%d", id);
 		else
-			res.name = remap[id];
+			res.name = remap[id - 1];
```

Comment:
Unfortunately, that now means that an ID of 0 would wrap here.

I think it would probably be best not to do any arithmetics on the ID, and simply make the remap vectors map directly by adding a dummy "" value at index 0 in the engine code for KotOR and KotOR2.

Or am I overlooking something here? @clone2727, you and me wrote the remapping in the first place like ages ago, no? Am I forgetting something? :P

Reply:
It broke in bb44be0. It previously always subtracted 1.

Yeah, could just put a dummy value at index 0.

Reply:
Yes, I think it breaking is my fault. I explictly called that out in the review because I didn't even remember.